package.json

```diff
@@ -54,7 +54,7 @@
     "sinon": "^13.0.0"
   },
   "peerDependencies": {
-    "ember-cli": "~3.2.0 || ^4.0.0"
```

Comment:
@NullVoxPopuli shouldn't this be `^3.2.0 || ^4.0.0`? I think there is no reason to prevent Ember CLI v3 users

Reply:
The `peerDependencies` entry was added a long time ago, but was never released until v3.3.0. 
Maybe the safest thing to do, is to completely remove it for now and then release a patch? 
We can always re-add it later and release a new major just to be sure?